testnet/stacks-node/src/config.rs

```diff
@@ -895,7 +899,7 @@ impl std::default::Default for Config {
     }
 }
 
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
```

Comment:
Why does this need `Serialize` and `Deserialize`?

Reply:
removed serialize